ob_v3p0/context.json

```diff
@@ -3,26 +3,76 @@
         "id" : "@id",
         "type" : "@type",
 
-        "obi" : "https://purl.imsglobal.org/spec/ob/vocab#",
+        "obi" : "https://w3id.org/openbadges#",
+        "clr": "https://purl.imsglobal.org/spec/clr/v1p0/context/clr_v1p0.html",
+        "cred": "https://www.w3.org/2018/credentials/#",
+        "dc": "http://purl.org/dc/terms/",
+        "schema": "http://schema.org/",
+        "sec": "https://w3id.org/security#",
         "xsd" : "http://www.w3.org/2001/XMLSchema#",
 
-        "assertion" : "obi:dtAssertion",
+        "Achievement": "clr:Achievement",
+        "Criteria": "obi:Criteria",
+        "Endorsement": "cred:VerifiableCredential",
```

Comment:
I think the IRI could be "obi:Endorsement" or "obi:EndorsementCredential". In the [vc-data-model examples context](https://www.w3.org/2018/credentials/examples/v1), the various credentials use the local prefix `ex` not `cred`. I'm guessing that is because the JSON `type` must include both "VerifiableCredential" and the specific type of credential represented. For example, `"type": ["VerifiableCredential", "AlumniCredential"]`.

I prefer adding "Credential" to the terms that are credentials. I saw this in all the vc-data-model examples and in the examples context file. It made it easier for me to know which things are credentials.